src/main/java/org/broadinstitute/hellbender/engine/filters/AlignmentAgreesWithHeaderReadFilter.java

```diff
@@ -9,7 +9,7 @@
 import java.io.Serializable;
 
 /**
- * Checks to ensure that the alignment of each read makes sense based on the contents of the header.
+ * Filter out reads where the alignment does not make sense based on the contents of the header.
```

Comment:
Can you add a few concrete examples of things that "don't make sense"? Right now this doesn't tell me what scenarios this filter would address.